test/test_manifest.json

```diff
@@ -1521,6 +1521,12 @@
        "lastPage": 1,
        "type": "eq"
     },
+    {  "id": "issue6387",
+       "file": "pdfs/issue6387.pdf",
+       "md5": "08c39ac6d0aab1596e6e59793eaf3ee4",
+       "rounds": 1,
+       "type": "eq"
+    },
```

Comment:
You'll have to add another block, since there isn't support for specifying multiple `type`s for one test.